pkg/data/task_result.go

```diff
@@ -35,8 +35,18 @@ type TranscodeTaskOutput struct {
 	Asset UploadTaskOutput `json:"asset,omitempty"`
 }
 
+type TranscodeFileTaskOutputMp4 struct {
+	Renditions []TranscodeFileTaskOutputPath `json:"renditions,omitempty"`
+}
+
+type TranscodeFileTaskOutputPath struct {
+	Path string `json:"path,omitempty"`
+}
+
 type TranscodeFileTaskOutput struct {
-	VideoFilePath string `json:"videoFilePath"`
+	BaseUrl string                      `json:"baseUrl,omitempty"`
+	Hls     TranscodeFileTaskOutputPath `json:"hls,omitempty"`
+	Mp4     TranscodeFileTaskOutputMp4  `json:"mp4,omitempty"`
```

Comment:
(yesterday I learned that) `omitempty` actually doesn't work if the field is a struct, so these need to be pointers instead if you want them actually omitted when not set
```suggestion
	Hls     *TranscodeFileTaskOutputPath `json:"hls,omitempty"`
	Mp4     *TranscodeFileTaskOutputMp4  `json:"mp4,omitempty"`
```

Reply:
Good point, thanks!